test/fixtures/remote-error-test-cases/default-messages.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const jstp = require('../../../');
+
+module.exports = Object.keys(jstp)
+  .filter(key => key.startsWith('ERR_'))
```

Comment:
@aqrln this test is for checking that every jstp error has a default message. Thus introspection is needed here in case someone adds new jstp error but forgets to add a default message.